Automatize the package publishment (#139)
meilisearch/meilisearch-java: CONTRIBUTING.md

Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/meilisearch-java/releases).

#### Sign your files and upload to Maven Repository manually <!-- omit in TOC -->

> 1. Prepare the environment by filling the environment variables needed with all the credentials:

Reviewer: Are this variables stored at `gradle.properties`? We can specify this on that step

Reviewer: Oh, I see this

> gradle.properties has been deleted and we now use environment variables

Is there a reason for removing `gradle.properties`?

Author: I don't store any variables in `gradle.properties` anymore because they are used as environment variables or in the command by the publishing workflow.
That's why I am wondering if we still need those steps in the CONTRIBUTING.MD. If so, we might have a specific manual process (using `gradle.properties`).

> 1. Prepare the environment by filling the environment variables needed with all the credentials:

Reviewer: ```suggestion
1. Set the environment variables listed below with the required credentials:
```

> ```

Reviewer: ```suggestion
```bash
```

> OSSRH_USERNAME=<maven-username>
> OSSRH_PASSWORD=<maven-password>

Reviewer: ```suggestion
export OSSRH_USERNAME=<maven-username>
export OSSRH_PASSWORD=<maven-password>
```

> SIGNINT_KEY_ID=<id-associated-to-the-gpg-key>
> SIGNING_PASSWORD=<passphrase-associated-to-the-gpg-key>
> SIGNING_SECRET_KEY_RING_FILE=<gpg-key-encoded-in-base64>

Reviewer: ```suggestion
export SIGNINT_KEY_ID=<id-associated-to-the-gpg-key>
export SIGNING_PASSWORD=<passphrase-associated-to-the-gpg-key>
export SIGNING_SECRET_KEY_RING_FILE=<gpg-key-encoded-in-base64>
```
